Loop/Settings/GeneralSettingsView.swift

```diff
@@ -180,17 +180,17 @@ struct GeneralSettingsView: View {
                 })
             }
 
-            Section("Accent Color") {
-                Toggle("Use System Accent Color", isOn: $useSystemAccentColor)
+            Section("Accent color") {
```

Comment:
Titles/headers should be in title case, meaning this would be "Accent Color"

Reply:
Updated.